statistics/histogram.go

```diff
@@ -1428,6 +1433,7 @@ func (idx *Index) expBackoffEstimation(sctx sessionctx.Context, coll *HistColl,
 	for i := 0; i < len(indexRange.LowVal); i++ {
 		tmpRan[0].LowVal[0] = indexRange.LowVal[i]
 		tmpRan[0].HighVal[0] = indexRange.HighVal[i]
+		tmpRan[0].Collators[0] = indexRange.Collators[i]
```

Comment:
Seems we should replace all usages of `Range.Collators[i]` with `Range.GetCollator(i)`?